app/models/work_package.rb

```diff
@@ -59,7 +59,7 @@ class WorkPackage < ApplicationRecord
   belongs_to :priority, class_name: "IssuePriority"
   belongs_to :category, class_name: "Category", optional: true
 
-  has_many :time_entries, dependent: :delete_all
+  has_many :time_entries, dependent: :delete_all, as: :entity
```

Comment:
Active Record Doctor reports "use `dependent: :destroy` or similar on WorkPackage.time_entries - associated model TimeEntry has callbacks that are currently skipped"

```suggestion
  has_many :time_entries, dependent: :destroy, as: :entity
```

Reply:
I did mot want to change anything regarding this logic, but you are correct. Let's see if this causes any weird side effects.